src/controller.rs

```diff
@@ -149,7 +149,7 @@ impl MetricsController {
     ///
     /// *event_label* -- Description of what the metric is. For example, &apos;memory&apos;
     ///
-    /// *event_value* -- Numeric value of the metric.
+    /// *event_value* -- Numeric value of the metric. which is Integer
```

Comment:
It would be better if comment was a full sentence. For example, "Numeric value of the metric, which is an Integer."